File: util/model/src/main/java/hu/radio/tilos/model/Mix.java
package hu.radio.tilos.model;

import hu.radio.tilos.model.type.MixCategory;
import hu.radio.tilos.model.type.MixType;

import javax.persistence.*;
import java.util.Date;

@Entity()
@Table(name = "mix")
public class Mix {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;


    @Basic
    @Column(length = 40)
    private String author;


    @Basic
    @Column(length = 160)
    private String title;


    @Basic
    @Column
    private String file;

    @ManyToOne
    @JoinColumn(name = "show_id")
    private Show show;

    @Temporal(TemporalType.DATE)
    @Column
    private Date date;

    @Column
    private MixType type;

    @Column
    private String content;

    @Column
    MixCategory category;

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public int getTypeCode(){
        return type.ordinal();
    }
    public MixCategory getCategory() {
        return category;
    }

    public void setCategory(MixCategory category) {
        this.category = category;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getFile() {
        return file;
    }

    public void setFile(String file) {
        this.file = file;
    }

    public Show getShow() {
        return show;
    }

    public void setShow(Show show) {
        this.show = show;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public MixType getType() {
        return type;
    }

    public void setType(MixType type) {
        this.type = type;
    }
}

File: backend/module/Radio/src/Radio/Entity/Mix.php
<?php
namespace Radio\Entity;

use Doctrine\ORM\Mapping as ORM,
    Radio\Permissions\Acl,
    Radio\Entity\Role;

/**
 * @ORM\Entity
 * @ORM\Table(name="mix")
 * */
class Mix
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue
     * */
    private $id;

    /**
     * @ORM\Column(type="string",length=160)
     */
    private $author;

    /**
     * @ORM\Column(type="string",length=160, nullable=false)
     */
    private $title;

    /**
     * @ORM\Column(type="string",length=255, nullable=false)
     */
    private $file;

    /**
     * @ORM\ManyToOne(targetEntity="Show", fetch="EAGER")
     */
    protected $show;

    /**
     * @ORM\Column(type="date", nullable=true)
     * */
    protected $date;

    /**
     * @ORM\Column(type="integer")
     * */
    protected $type = 0;

    /**
         * @ORM\Column(type="integer")
         * */
        protected $category = 0;

    /**
     * @return mixed
     */
    public function getAuthor()
    {
        return $this->author;
    }

    /**
     * @param mixed $author
     */
    public function setAuthor($author)
    {
        $this->author = $author;
    }

    /**
     * @return mixed
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param mixed $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }

    /**
     * @return mixed
     */
    public function getFile()
    {
        return $this->file;
    }

    /**
     * @param mixed $file
     */
    public function setFile($file)
    {
        $this->file = $file;
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getShow()
    {
        return $this->show;
    }

    /**
     * @param mixed $show
     */
    public function setShow($show)
    {
        $this->show = $show;
    }

    /**
     * @return mixed
     */
    public function getTitle()
    {
        return $this->title;
    }

    /**
     * @param mixed $title
     */
    public function setTitle($title)
    {
        $this->title = $title;
    }

    /**
     * @return mixed
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param mixed $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }


}